library/Haste/Ajax/ReloadHelper.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+namespace Haste\Ajax;
+
+use Haste\Http\Response\JsonResponse;
+
+class ReloadHelper
+{
+    const TYPE_CONTENT_ELEMENT = 'ce';
+    const TYPE_FRONTEND_MODULE = 'fmd';
```

Comment:
I would prefer `TYPE_CONTENT` and `TYPE_MODULE` (because it's shorter and follows the table name)